text/warn/inappropriate.md

```diff
@@ -0,0 +1,6 @@
+You've violated the following rule(s):
+
+- Treat everyone with respect. Absolutely no harassment, witch hunting, sexism, racism, or hate speech will be tolerated.
+- No NSFW or obscene content. This includes text, images, or links featuring nudity, sex, hard violence, or other graphically disturbing content.
```

Comment:
Should we split this up into two messages? The two points are kinda unrelated and just makes for. I propose `conduct` and `content` for the identifiers.

Reply:
Yeah, I agree with splitting.
For identifiers, I want something descriptive for others to understand what the warning was about. These messages are DMed to the target user.

Reply:
Actually, `conduct` and `content` might be nice and short.